src/CMakeLists.txt

```diff
@@ -1,8 +1,19 @@
 find_package(PkgConfig REQUIRED)
-pkg_check_modules(EventdEvent REQUIRED libeventd-event>=0.3)
+pkg_check_modules(EventdEvent REQUIRED libeventd>=0.3)
 pkg_check_modules(EventdPlugin REQUIRED libeventd-plugin>=0.3)
 pkg_check_modules(SystemdJournal REQUIRED libsystemd-journal)
 
+macro(pkg_config_variable PKG VAR OUT)
+    execute_process(COMMAND ${PKG_CONFIG_EXECUTABLE} --variable=${VAR} ${PKG}
+        OUTPUT_VARIABLE ${OUT})
+    string(STRIP ${${OUT}} ${OUT})
```

Comment:
It'd be right :). `LIB_SUFFIX` should be `/<arch>` on Debian. Most of the weirdness of CMake on *nix comes from the fact that CMake supports Windows and OS X (for all their warts) while autotools blithely ignores the fact that most of the world isn't *nix (much to my dismay as well). For example, on Windows, there are *two* places libraries need to know where to be installed to: one for the `.dll` bit (usually `bin`) and the other for the `.lib` bit (usually `lib`). Then you have the joys of *Python* on Windows where you put your modules under `bin/Lib/site-packages` rather than something…sensible.